### PR TITLE
bugfix/apptoken: missing app token verification codes

### PR DESCRIPTION
**Pull request by 1046102779**
add: external client rpc MicroLogic server with app token verification logic.

### CODE REVIEW

File: service/http/invoke_test.go [hunk 1]
```diff
@@ -20,6 +21,41 @@ func TestInvocationHandlerWithoutHandler(t *testing.T) {
...
+	// forbbiden.
+	req, err := http.NewRequest(http.MethodPost, "/", strings.NewReader(data))
+	assert.NoErrorf(t, err, "error creating request")
+	req.Header.Set("Content-Type", "application/json")
+
+	resp := httptest.NewRecorder()
+	s.mux.ServeHTTP(resp, req)
+	assert.Equal(t, http.StatusNonAuthoritativeInfo, resp.Code)
+
+	// pass.
```
Comment (daixiang0) [inline service/http/invoke_test.go:51]: Please add false cases.

File: service/http/service.go [hunk 2]
```diff
@@ -36,11 +37,13 @@ func newServer(address string, router *mux.Router) *Server {
 		},
 		mux:                router,
 		topicSubscriptions: make([]*common.Subscription, 0),
+		authToken:          os.Getenv(common.AppAPITokenEnvVar),
```
Comment (daixiang0) [inline service/http/service.go:40]: Store token in env?

Comment (1046102779) [reply service/http/service.go:40]: Yes, Need to provide other way, such as function?

File: service/http/invoke_test.go (hunk 1 above)
Comment (1046102779) [reply service/http/invoke_test.go:51]: false: http.StatusNonAuthoritativeInfo
true: http.StatusOK

**Comment by daixiang0** [comment]
@1046102779 please fix CI issues.

**Comment by 1046102779** [comment]
@daixiang0  how to solve it?
image

**Comment by daixiang0** [comment]
It is what ths issue exist: https://github.com/dapr/go-sdk/runs/4640394565?check_suite_focus=true#step:8:5022

**Comment by 1046102779** [comment]
> exist

fixed.

**Comment by 1046102779** [comment]
> It is what ths issue exist: https://github.com/dapr/go-sdk/runs/4640394565?check_suite_focus=true#step:8:5022

fixed.

**Comment by yaron2** [comment]
@daixiang0 are you ok with this PR? it looks good to me.

**Comment by daixiang0** [comment]
@1046102779 please fix DCO issue. LGTM overall.

**Comment by 1046102779** [comment]
https://github.com/dapr/go-sdk/pull/236

**Pull request closed without merging**